{"name": "test_divide_number", "status": "skipped", "statusDetails": {"message": "XFAIL Need to investigate\n\n_pytest.outcomes.XFailed: Need to investigate", "trace": "self = <DemoThirdTest.test_xfail.TestMath object at 0x000002A2379ED810>\n\n    def test_divide_number(self):\n>       pytest.xfail(reason=\"Need to investigate\")\nE       _pytest.outcomes.XFailed: Need to investigate\n\nDemoThirdTest\\test_xfail.py:7: XFailed"}, "start": 1754971611415, "stop": 1754971611415, "uuid": "cd3adf4e-9e69-4322-b4b5-c0c3bf569222", "historyId": "39b8d5685e3dd091983cb604f7544840", "testCaseId": "39b8d5685e3dd091983cb604f7544840", "fullName": "DemoThirdTest.test_xfail.TestMath#test_divide_number", "labels": [{"name": "parentSuite", "value": "DemoThirdTest"}, {"name": "suite", "value": "test_xfail"}, {"name": "subSuite", "value": "TestMath"}, {"name": "host", "value": "DESKTOP-0LJMU54"}, {"name": "thread", "value": "19604-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "DemoThirdTest.test_xfail"}], "titlePath": ["DemoThirdTest", "test_xfail.py", "TestMath"]}